### PR TITLE
Update Replacer and Importurls

### PR DESCRIPTION
**Pull request by thc202**
Update Replacer API, per release of version 7.
Add description to Importurls API endpoint.

### CODE REVIEW

**Comment by thc202** [comment]
Rebased, build passing.

File: src/zapv2/replacer.py [hunk 1]
```diff
@@ -35,12 +35,14 @@ def rules(self):
...
         """
         Adds a replacer rule. For the parameters: desc is a user friendly description, enabled is true or false, matchType is one of [REQ_HEADER, REQ_HEADER_STR, REQ_BODY_STR, RESP_HEADER, RESP_HEADER_STR, RESP_BODY_STR], matchRegex should be true if the matchString should be treated as a regex otherwise false, matchString is the string that will be matched against, replacement is the replacement string, initiators may be blank (for all initiators) or a comma separated list of integers as defined in <a href="https://github.com/zaproxy/zaproxy/blob/develop/src/org/parosproxy/paros/network/HttpSender.java">HttpSender</a>  
         This component is optional and therefore the API will only work if it is installed
         """
-        params = {'description': description, 'enabled': enabled, 'matchType': matchtype, 'matchRegex': matchregex, 'matchString': matchstring, 'replacement': replacement, 'apikey': apikey}
+        params = {'description': description, 'enabled': enabled, 'matchType': matchtype, 'matchRegex': matchregex, 'matchString': matchstring, 'apikey': apikey}
+        if replacement is not None:
```
Comment (Kilo59) [inline src/zapv2/replacer.py]: Does this work?

```suggestion
        if not replacement:
```

This condition would be met if `replacement` evalutes to `False` or `None`. I assume that would work but I don't have enough context about how it gets used.

Comment (thc202) [reply src/zapv2/replacer.py]: That would change the desired behaviour, we want to send the parameter whenever it has a value (even if false).

**Comment by psiinon** [comment]
@kingthorin review? :)

**Pull request merged**